rpc/src/services/protocol/proto_005_2/operations_service.rs

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) SimpleStaking and Tezedge Contributors
+// SPDX-License-Identifier: MIT
+
+// NOTE: made a separate file, the operatins RPC will encapsulate more functionality,
+//       for now, it is just one function, that fetches the operations
+
+use failure::bail;
+
+use storage::{BlockStorage, BlockStorageReader};
+use storage::persistent::PersistentStorage;
+
+use crate::helpers::{get_block_hash_by_block_id, BlockOperations};
+use crate::rpc_actor::RpcCollectedStateRef;
+
+pub(crate) fn get_operations(_chain_id: &str, block_id: &str, persistent_storage: &PersistentStorage, state: &RpcCollectedStateRef) -> Result<Option<BlockOperations>, failure::Error> {
```

Comment:
@adonagy 
this is not using anything special, by protocol,
put it in rpc/src/services/protocol/mod.rs for every protocol like vote/listing

Reply:
Whoops, forgot about that. I'll put it in place